code/modules/client/verbs/ooc.dm

```diff
@@ -414,7 +414,7 @@ GLOBAL_VAR_INIT(normal_ooc_colour, "#002eb8")
 
 /// Attempt to automatically fit the viewport, assuming the user wants it
 /client/proc/attempt_auto_fit_viewport()
-	if (!prefs.read_preference(/datum/preference/toggle/auto_fit_viewport))
+	if(!prefs.read_preference(/datum/preference/toggle/auto_fit_viewport) || prefs?.read_preference(/datum/preference/toggle/widescreen))
 		return
 	if(fully_created)
 		INVOKE_ASYNC(src, .verb/fit_viewport)
```

Comment:
this is somewhat unfortunate, auto fit viewport essentially tells the game to set the viewport as large as our monitor and settings allows, sad to make that not supported in this new format

Reply:
Aye, I can fix it up to auto determine the scale for whats best instead of ignore it